.github/workflows/validate.yml

```diff
@@ -18,3 +20,20 @@ jobs:
       run: pip install codespell
     - name: run codespell
       run: codespell
+  lint:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version: 1.x # latest stable
+    - name: install deps
+      run: |
+        sudo apt-get -qq update
+        sudo apt-get -qq install libseccomp-dev libdevmapper-dev
+    - name: lint
+      uses: golangci/golangci-lint-action@v3
+      with:
+        version: "${{ env.LINT_VERSION }}"
+    - name: validate seccomp
+      run: ./tools/validate_seccomp.sh ./pkg/seccomp
```

Comment:
We already run this as part of the normal cirrus job so this is redundant.

Reply:
Yes, I am aware of that (this is where I copied the code from). I guess I did that since I was thinking about removing this step (running the linter) from the cirrus job (but have not done it yet).

Reply:
Lets do this in a separate PR